### PR TITLE
Update contributing templates

### PR DESCRIPTION
**Pull request by tchaton**
## What does this PR do?

Fixes #<issue_number>

## Before submitting
- [ ] Was this discussed/approved via a GitHub issue? (not for typos and docs)
- [ ] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [ ] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [ ] Did you make sure to update the documentation with your changes? (if necessary)
- [ ] Did you write any new necessary tests? (not for typos and docs)
- [ ] Did you verify new and existing tests pass locally with your changes?
- [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)
- [ ] Did you list all the breaking changes introduce by this pull request ?

## PR review
Anyone in the community is free to review the PR once the tests have passed.
Before you start reviewing make sure you have read [Review guidelines](https://github.com/PyTorchLightning/pytorch-lightning/wiki/Review-guidelines). In short, see the following bullet-list:

 - [ ] Is this pull request ready for review? (if not, please submit in draft mode)
 - [ ] Check that all items from **Before submitting** are resolved
 - [ ] Make sure the title is self-explanatory and the description concisely explains the PR
 - [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?
Make sure you had fun coding 🙃

### Learn more about our other projects:

- [Flash](https://github.com/PyTorchLightning/lightning-flash): Collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
- [Bolts](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks and more for research and production with PyTorch Lightning and PyTorch
- [Metrics](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.
- [Lightning Transformers](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -54,3 +54,11 @@ python collect_env_details.py
...
 
 <!-- Add any other context about the problem here. -->
+
+
+### :star: If you liked PyTorch Lightning, you might also like :star:
+
+- [Flash](https://github.com/PyTorchLightning/lightning-flash): Collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
+- [Bolts](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks and more for research and production with PyTorch Lightning and PyTorch
+- [Metrics](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.
+- [Lightning Transformers](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.
```
Comment (kaushikb11) [inline .github/ISSUE_TEMPLATE/bug_report.md:64]:
```suggestion
```
@tchaton It shouldn't be in `bug_report.md`

**Review by kaushikb11 — APPROVED** [review]
LGTM!

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -19,6 +23,7 @@ Fixes #<issue_number>
 - [ ] Did you write any new necessary tests? (not for typos and docs)
 - [ ] Did you verify new and existing tests pass locally with your changes?
 - [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)
+- [ ] Did you list all the breaking changes introduce by this pull request ?
```
Comment (Borda) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: is this needed while there are people who do not border with filling the checklist at all?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -54,3 +54,12 @@ python collect_env_details.py
...
 <!-- Add any other context about the problem here. -->
+
+
+
+### Learn more about our other projects:
+
+- [Flash](https://github.com/PyTorchLightning/lightning-flash): Collection of tasks for fast prototyping, baselining, finetuning and solving problems with deep learning
+- [Bolts](https://github.com/PyTorchLightning/lightning-bolts): Pretrained SOTA Deep Learning models, callbacks and more for research and production with PyTorch Lightning and PyTorch
+- [Metrics](https://github.com/PyTorchLightning/metrics): Machine learning metrics for distributed, scalable PyTorch applications.
+- [Lightning Transformers](https://github.com/PyTorchLightning/lightning-transformers): Flexible interface for high performance research using SOTA Transformers leveraging Pytorch Lightning, Transformers, and Hydra.
```
Comment (Borda) [inline .github/ISSUE_TEMPLATE/bug_report.md:65]: I do not think it is the right place for promo

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (tchaton) [reply .github/PULL_REQUEST_TEMPLATE.md:26]: Yes, I believe it is needed.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 3 above)
Comment (tchaton) [reply .github/ISSUE_TEMPLATE/bug_report.md:65]: Curious to hear the team thoughts there. I think the Lighting Ecosystem is worth mentioning, especially for new users.

Comment (Borda) [reply .github/ISSUE_TEMPLATE/bug_report.md:65]: yes, but here you mentioning it for all users, not new users...
for new users we had this bot - https://github.com/PyTorchLightning/pytorch-lightning/blob/release/1.0.x/.github/workflows/greetings.yml

**Review by SeanNaren — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (ethanwharris) [inline .github/PULL_REQUEST_TEMPLATE.md:26]: Small nit follow-up 😆 
```suggestion
- [ ] Did you list all the breaking changes introduced by this pull request ?
```

**Review by ethanwharris — APPROVED** [review]

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -19,6 +23,7 @@ Fixes #<issue_number>
 - [ ] Did you write any new necessary tests? (not for typos and docs)
 - [ ] Did you verify new and existing tests pass locally with your changes?
 - [ ] Did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning/blob/master/CHANGELOG.md)? (not for typos, docs, test updates, or internal minor changes/refactorings)
+- [ ] Did you list all the breaking changes introduced by this pull request ?
```
Comment (kaushikb11) [inline .github/PULL_REQUEST_TEMPLATE.md:26]:
```suggestion
- [ ] Did you list all the breaking changes introduced by this pull request?
```

**Review by awaelchli — APPROVED** [review]

**Review by SkafteNicki — APPROVED** [review]

**Review by williamFalcon — APPROVED** [review]

**Review by edenlightning — APPROVED** [review]

**Review by SeanNaren — APPROVED** [review]
